lib/discourse.rb

```diff
@@ -204,6 +204,22 @@ def self.unofficial_plugins
     plugins.find_all { |p| !p.metadata.official? }
   end
 
+  def self.find_plugins(args)
+    plugins.find_all do |plugin|
+      next if args[:include_official] == false && plugin.metadata.official?
+      next if args[:include_unofficial] == false && !plugin.metadata.official?
+      next if args[:include_disabled] != true && !plugin.enabled?
```

Comment:
Using equality is slightly easier to read than inequality

```suggestion
      next if args[:include_disabled] == false && !plugin.enabled?
```